vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/embedding_info.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_PIPELINE_TEXT_PROCESSING_EMBEDDING_INFO_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_PIPELINE_TEXT_PROCESSING_EMBEDDING_INFO_H_
+
+#include <string>
+
+#include "bat/ads/internal/ml/data/vector_data.h"
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+struct TextEmbeddingInfo final {
+  TextEmbeddingInfo();
+  TextEmbeddingInfo(const TextEmbeddingInfo& info);
+  TextEmbeddingInfo& operator=(const TextEmbeddingInfo& info);
+  ~TextEmbeddingInfo();
+
+  std::string text;
+  std::string text_hashed;
```

Comment:
`text_base64`

Reply:
@tmancey using `text_base64` is a bit misleading here, since it is the base_64 encoding of the sha_256 hash of the text. I'm okay calling it `text_hashed_base64` if you feel that is better than `text_hashed`?

Reply:
your thoughts on `hashed_text_base64`?